src/Microsoft.AspNetCore.App/build/netcoreapp2.1/Microsoft.AspNetCore.App.targets

```diff
@@ -0,0 +1,7 @@
+<Project InitialTargets="EnsureTFMCompatibility">
+  <Target Name="EnsureTFMCompatibility">
+    <Error
+      Text="This version of Microsoft.AspNetCore.App is only compatible with the netcoreapp2.1 target framework."
```

Comment:
Have we considered expanding the error message to say how to resolve this?

Reply:
I'll add one.